docs/user/gcp/customization.md

```diff
@@ -0,0 +1,106 @@
+# GCP Platform Customization
+
+Beyond the [platform-agnostic `install-config.yaml` properties](../customization.md#platform-customization), the installer supports additional, GCP-specific properties.
+
+* `projectID` (required string): The project where the cluster should be created.
+* `region` (required string): The GCP region where the cluster should be created.
+* `network` (optional string): The ID of an existing GCP VPC where the cluster infrastructure should be provisioned.
```

Comment:
is it really an ID like `/project/<>/network/<>` or is it only the name of the network?

Reply:
it is a name. fixed